src/main.js

```diff
@@ -3,6 +3,18 @@
 const electron = require('electron');
 const app = electron.app; // Module to control application life.
 
+var cmd = process.argv[1];
+if (cmd === '--squirrel-uninstall') {
```

Comment:
First, how about checking the part which is working properly? Sure, the debugging of installation seems difficult, but for example, you can inspect by writing simple log file with `fs` module.